src/components/report-problem-comment/styles.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+import { StyleSheet } from 'react-native';
+
+export default StyleSheet.create({
+
+largeimage: {
+	flex:1,
+	borderWidth:1
+},
+textInputWrapper: {
+	justifyContent:'center',
+	paddingTop:50,
+	flex:1,
+	paddingLeft:10,
+	paddingRight:10,
+	paddingBottom:30
+},
+textInput :{
+	flex:1,
+	borderColor: 'black',
```

Comment:
Colors should be fed in from our color style file in the global directory. So either use an existing color value or add to the list of colors there